### ISSUE TITLE
FtCLI.exit(Exit) creates an unnecessary custom Object

### ISSUE THREAD

**Issue by simonjenga** [yegor256/takes#427]
In the current implementation of the method ```org.takes.http.FtCLI.exit(Exit)``` , 
if  the line ```exit.ready() || custom.ready();``` evaluates to ```true```, we will have already incurred the additional overhead costs (computing time & resources) of creating a custom ```Object``` that was not necessary to do so in the beginning since it will never used!

The current method implementation needs to be changed to be as follows:

```
private Exit exit(final Exit exit) {
	final Exit toExit;
	if (exit.ready()) {
		toExit = Exit.ALWAYS;
	} else {
		final long start = System.currentTimeMillis();
		final long max = this.options.lifetime();
		toExit = new Exit() {
			@Override
			public boolean ready() {
				return (System.currentTimeMillis() - start) > max;
			}
		};
	}
	return toExit;
}
```

The following ```Object``` is then added to the interface ```org.takes.http.Exit```

```
/**
 * Always exit.
 */
final Exit ALWAYS = new Exit() {
	@Override
	public boolean ready() {
		return true;
	}
};
```

**Comment by davvd** [comment]
@yegor256 please do something about it, see [par.21](http://www.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@simonjenga I believe it's fixed in e4848a7, thanks
